kvm/src/lib.rs

```diff
@@ -1482,4 +1508,80 @@ mod tests {
         let kvm = Kvm::new().unwrap();
         assert_eq!(kvm.get_api_version(), KVM_API_VERSION as i32);
     }
+
+    #[test]
+    fn test_cpu_id_new() {
+        let cpuid = CpuId::new(5);
+
+        let expected_len = 208;
+        assert!(cpuid.bytes.len() == expected_len);
+
+        let mut expected_bytes = vec![0; expected_len];
+        expected_bytes[0] = 5;
+        assert!(cpuid.bytes == expected_bytes);
+    }
+
+    #[test]
+    fn test_cpu_id_mut_entries_slice() {
+        let mut cpuid = CpuId::new(5);
```

Comment:
Lots of hardcoded offsets in these tests, that make it difficult to understand. I suggest a combination of computing the values, using consts, and adding comments, so the whole thing becomes more digestible.

Reply:
I added some comments and replaced some of the hardcoded values with variables.

I think the safest thing is to keep some of the hardcoded expected values. I wouldn't compute them. Otherwise we would risk making the same mistakes in the tests as in the actual implementation.